Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM node:16
+FROM node:19
 
+ENV TERM=xterm
+RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 COPY package*.json ./
-
 RUN npm install
 
-COPY . .
-
+COPY tsconfig.json ./
+COPY src ./src
```

Comment:
@carlthelooper I think we also need to COPY .env ./ right?

Reply:
tho you can also use this dockerfile wihtout compose... just use docker build . 
i'd say we make sure this file works anyways

Reply:
To stay consistent for spinning up a container with docker-compose or docker you'd need to leave copying the .env out of the dockerfile.  For spinning up the container with docker you'd just add a param for including the `env_file` or including all the env variables if you really wanted to.